app/services/commonApiService.ts

```diff
@@ -56,9 +56,7 @@ export const updateIncrementalSyncDT = async(response: any, dispatch: any, _navi
   const articleresp = response.find((y:any)=>y.apiEndpoint == appConfig.articles);
```

Comment:
“Can we improve the variable name ‘artilceresp’? Also, can we adhere to a single formatting scheme?”